packages/client/src/sound-board/service/soundboard-service.ts

```diff
@@ -1,5 +1,6 @@
+const AudioClient = new Audio();
```

Comment:
It's here because it would be nice to have just one instance of the new Audio

Reply:
This is fine :) But I think it should be lower case

```suggestion
const audioClient = new Audio();
```